FinSearchUnified/Tests/Subscriber/FrontendTest.php

```diff
@@ -74,6 +68,16 @@ public function listingCountConditionProvider()
         ];
     }
 
+    public function vendorProvider()
+    {
+        return [
+            'setRequestUri contains vendor=Brand' => ['Brand'],
```

Comment:
Descriptions of data sets shouldn't be too technical. Change these to:
* `Regular vendor name`
* `Vendor name containing whitespace`
* `Vendor name containing "+" character`
* `Vendor name containing special characters`